### DISCUSSION TITLE
Python prerequisites for onnxruntime 1.12.x

### DISCUSSION THREAD

**Discussion by ai2ys** [microsoft/onnxruntime#13320] [Other Q&A] [answered]
When trying to install onnxruntime inside a Docker container using Python 3.6.9 (pip 21.3.1) I got the following error when trying to install version 1.12.0.

```
pip install onnxruntime==1.12.0
ERROR: Could not find a version that satisfies the requirement onnxruntime==1.12.0 (from versions: 0.1.2, 0.1.3, 0.1.4, 0.2.1, 0.3.0, 0.4.0, 0.5.0, 1.0.0, 1.1.0, 1.1.1, 1.1.2, 1.2.0, 1.3.0, 1.4.0, 1.5.1, 1.5.2, 1.6.0, 1.7.0, 1.8.0, 1.8.1, 1.9.0, 1.10.0)
ERROR: No matching distribution found for onnxruntime==1.12.0
```

Is there an overview of prerequisites for the specific onnxruntime versions? 
What is the minimum required Python version for onnxruntime 1.12.0?

**Comment by snnn** [comment, marked as answer]
> What is the minimum required Python version for onnxruntime 1.12.0?

3.7. As python 3.6 is already EOL.
